src/Phansible/Roles/Xdebug.php

```diff
@@ -34,4 +36,17 @@ public function getInitialValues()
           'settings' => [],
         ];
     }
+
+    public function transformValues(array $values, VagrantBundle $vagrantBundle)
+    {
+        if ($values['install'] == 1) {
+            $values['settings'] = [
```

Comment:
In my opinion passing the variables through the form would only give a benefit when we could edit them. 
That would result in GUI changes: something I wanted to prevent in this first step as the focus was to make xdebug usable again. In addition, I would question if it is really necessary to be able to edit the xdebug settings as it is at a detail level not a lot of people have the need to change, I could imagine. 

=> Did not know about the background of the `RoleValuesTransformer`, but it seemed to me as the right mechanism cause it transforms one single value selected in GUI (xdebug on/off) to a set of values used by provisioning.